src/main/scala/com/campudus/tableaux/database/model/tableaux/RowModel.scala

```diff
@@ -548,7 +549,8 @@ class UpdateRowModel(val connection: DatabaseConnection) extends DatabaseQuery w
          |  user_table_annotations_$tableId.row_id = subquery.row_id AND
          |  user_table_annotations_$tableId.column_id = subquery.column_id AND
          |  user_table_annotations_$tableId.type = subquery.type AND
-         |  user_table_annotations_$tableId.value = subquery.value
+         |  user_table_annotations_$tableId.value = subquery.value AND
+         |  user_table_annotations_$tableId.type = 'flag'
```

Comment:
Two lines above there is a check for "user_table_annotations_$tableId.type = subquery.type AND".
So the type has to be "flag" or this will always be false.
So line 551 can be deleted, as it has no effect anymore?

Reply:
right, flags must be unique for each cell, comment values may occur more than once in a cell.